project_resource_calendar/__manifest__.py

```diff
@@ -19,12 +19,12 @@
         'python': [],
     },
     'data': [
-        'views/room_type.xml',
```

Comment:
@eilst: Order the contents  based on the folder and file types

Reply:
I think this is done, it's ordered by views and then the acls

Reply:
Done, changed the order..if that's what you meant